.ci/util.sh

```diff
@@ -7,9 +7,21 @@ removeFolderWithProtectedFiles() {
 }
 
 function getCheckstylePomVersion {
-  echo "$(mvn -e --no-transfer-progress -q -Dexec.executable='echo' \
-                      -Dexec.args='${project.version}' \
-                      --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)"
+  CS_VERSION_FILE='/tmp/cs_version_file'
+  mvn -e --no-transfer-progress -q -Dexec.executable='echo' \
+                        -Dexec.args='${project.version}' \
+                        --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec \
+                         > $CS_VERSION_FILE
+  # checks last command error code
+  if [[ $? != 0 ]]
```

Comment:
We usually execute in mode "set -e" ( fail fast), so if "mvn" command failing it will stop execution of method/script so this IF is not going to happen.

Workaround is:
```
set +e
mvn ...
set -e
```

Reply:
@strkkk , ping